content/en/docs/concepts/storage/storage-capacity.md

```diff
@@ -0,0 +1,134 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+- pohly
+title: Storage Capacity
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+Storage capacity is limited and may vary depending on the node on
+which a pod runs: network-attached storage might not be accessible by
+all nodes, or storage is local to a node to begin with.
+
+{{< feature-state for_k8s_version="v1.19" state="alpha" >}}
+
+This page describes how Kubernetes keeps track of storage capacity and
+how the scheduler uses that information to schedule Pods onto nodes
+that have access to enough storage capacity for the remaining missing
+volumes. Without storage capacity tracking, it is random whether the
+selected node has enough storage for the Pod and multiple scheduling retries may be
+needed.
+
+Tracking storage capacity is supported for [{{< glossary_tooltip
+text="Container Storage Interface" term_id="csi" >}} (CSI) drivers](https://kubernetes-csi.github.io/docs/drivers.html) and
```

Comment:
nit: Did the link work out as expected? The tooltip is working.
I clicked on `CSI` and was forwarded to `https://deploy-preview-21634--kubernetes-io-vnext-staging.netlify.app/docs/concepts/storage/volumes/#csi`

Reply:
Indeed, it looks like the tooltip overrides the link destination. I've dropped the link. We can add it back under "enabling" once that table has information about drivers that support the feature (assuming that we consider it worth calling out there).